bluetooth/c/Dockerfile

```diff
@@ -22,8 +22,14 @@ ENV IMAGE_ARCH ${IMAGE_ARCH}
 # __deps__
 RUN apt-get -q -y update && \
     apt-get -q -y install \
+    bluez \
+    libc6-dev \
+    libbluetooth-dev \
 # DOES NOT REMOVE THIS LABEL: this is used for VS Code automation
     # __torizon_packages_dev_start__
+	bluez:arm64 \
```

Comment:
Also here this values can be removed, as the values listed on the `torizonPackages.json` get added automatically by the extension here